README.md

```diff
@@ -210,46 +210,31 @@ npm run dev
 
 Once the server starts, visit http://localhost:4321/ in your favourite browser. This runs with a hot reload which means any changes you make in the website and the documentation can be instantly viewed on the browser.
 
-## To build and deploy
-
-```bash
-cd docs
-npm run build
-```
-
 ### Docs directory structure
 
 1. `docs/src/content/docs/commands` is where all the commands are documented
 2. `docs/src/content/docs/tutorials` is where all the tutorials are documented
 
-## The story
+## The Story
 
-DiceDB started as a re-implementation of Redis in Golang and the idea was to - build a DB from scratch and understand the micro-nuances that come with its implementation. The database does not aim to replace Redis, instead, it will fit in and optimize itself for multicore computations running on a single-threaded event loop.
+DiceDB began as a re-implementation of Redis in Golang, with the goal of building a Database-as-a-Service optimized for modern hardware and real-time reactivity. While DiceDB does not aim to replace Redis, it specializes in multicore computations while running on a single-threaded event loop, making it ideal for building and scaling truly real-time applications.
```

Comment:
Not sure about wording on this.
cc: @JyotinderSingh

Reply:
```suggestion
DiceDB started as a re-implementation of Redis in Golang with the idea of building a DB from scratch to understand the micro-nuances that come with its implementation. DiceDB isn’t just another database; it’s a platform purpose-built for the real-time era. As real-time systems become increasingly prevalent in modern applications, DiceDB’s hyper-optimized architecture is positioned to power the next generation of user experiences.
```